backend/zero_alloc_checker.ml

```diff
@@ -2264,7 +2296,12 @@ end = struct
       | Iexit _ ->
         report t next ~msg:"transform" ~desc:"iexit" i.dbg;
         next
-      | Iifthenelse _ | Iswitch _ -> next
+      | Iifthenelse _ ->
+        report t next ~msg:"transform" ~desc:"ifthenelse" i.dbg;
+        next
+      | Iswitch _ ->
+        report t next ~msg:"transform" ~desc:"switch" i.dbg;
+        next
       | Icatch (_rc, _ts, _, _body) ->
         report t next ~msg:"transform" ~desc:"catch" i.dbg;
         next
```

Comment:
(Does not really matter, but the `desc` are
a bit inconsistent: `iexit`, `ifthenelse`,
and `try-with`.)